inc/saiswitch.h

```diff
@@ -409,6 +409,17 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_MAX_TEMP,
 
+    /**
+     * @brief The average value of the temperature
+     * retrieved from the switch sensors
```

Comment:
does this mean from all available sensors? not sure if description is good enough, if you have 1 sensor with 10 degree, and 2nd with 100 then average is 55 which maybe ok but system is defenetly not ok at 100

Reply:
updating description should be fine